js/file.js

```diff
@@ -0,0 +1,52 @@
+let books = JSON.parse(localStorage.getItem('cataloge'));
+const file = document.querySelector('.book');
+
+function addBooks(ti, aut) {
+  books.push({ title: ti, author: aut, id: books.length });
+  window.localStorage.setItem('cataloge', JSON.stringify(books));
+}
+
+/* eslint-disable */
```

Comment:
- Implement a well use of your linter. Try to use the right rule without disable eslint.